lib/utils/validators.js

```diff
@@ -0,0 +1,42 @@
+class Validator {
+  constructor(test, errorMessage) {
+    this.test = test || false;
+    this.errorMessage = errorMessage || '';
+  }
+
+  _validate(input) {
+    if (!this.test) {
+      return false;
+    }
+    return this.test.test(input);
+  }
+
+  isValid(input) {
+    return this._validate(input);
+  }
+}
+
+
+const none = new Validator(false);
+const email = new Validator(/.+@.+\./, 'Not a valid email address');
+const hasCapital = new Validator(/[A-Z]/, 'Must contain a capital letter');
+const isLowercase = new Validator(/^[A-Z]/, 'Must be all lowercase');
+const zipCode = new Validator(/\d{5}/, 'Enter a 5-digit ZIP Code');
```

Comment:
We should also support the 4-digit extension. So the pattern would look like `/\d{5}(?:-\d{4})?/`.